nikola/plugins/task/archive.py

```diff
@@ -97,10 +97,20 @@ def get_classification_friendly_name(self, classification, lang, only_last_compo
         elif len(classification) == 1:
             return classification[0]
         elif len(classification) == 2:
-            return nikola.utils.LocaleBorg().get_month_name(int(classification[1]), lang)
+            month = nikola.utils.LocaleBorg().get_month_name(int(classification[1]), lang)
+            if only_last_component:
+                return month
+            else:
+                year = classification[0]
+                return self.site.MESSAGES[lang].get('{month} {year}', '{month} {year}').format(year=year, month=month)
```

Comment:
You don’t really need .get(), `[]` will do it because you are allowed to assume all locales are correct.